### ISSUE TITLE
BUG - sev-2 - All - In-app recruitment has incorrect screenreader text

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9209]
## What happened?

Since #6151, in-app recruitment has had the main content of the screen be read out completely incorrectly (in what I'm assuming was a copy-paste issue never caught during initial implementation).

The first two of the three total bullet points on screen are (instead of their actual content) announcing messaging help text "If you think your life or health is in danger, call 911 or go to the nearest emergency room". The third one (screenshot below) is correctly reading out the content on the screen.

<img width="367" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM0VZQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0bcfbc3246fd8477f75cdfd915f54af272615f84/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

The problem is (very obviously) lines [67] and 71 of the in app recruitment screen. Should be a quick fix.
<img width="1594" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2dZQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3b922f5e1b87e2bc4f9ba6ba362f6b3306432883/image.png" alt="image.png" />

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
Confirmed on iOS and Android, we're now reading out what's showing on the screen (instead of telling people to call 911) for the first and second bullet points for in-app recruitment screen.

<img width="359" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOHdkQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d5127c23abdd5fb69421953afbd81e5398f2b473/image.png" alt="image.png" />

**Comment by TKDickson** [comment]
Doesn't require further testing in develop; closing
